modules/ldap_contacts/modules.php

```diff
@@ -776,7 +776,7 @@ function fetch_ldap_contacts($config, $user_config, $contact_store, $session=fal
     $ldap_config = ldap_add_user_auth($ldap_config, $user_config->get('ldap_contacts_auth_setting', array()));
     if (count($ldap_config) > 0) {
         foreach ($ldap_config as $name => $vals) {
-            $vals['name'] = $name;
+            is_array($vals) && $vals['name'] = $name;
```

Comment:
This is being fixed in this commit https://github.com/cypht-org/cypht/pull/869. You can remove this change.